src/app/select-role/error.jsx

```
@@ -1,18 +1,23 @@
 "use client";
 
+import { Button } from "@/components/common/Button";
+
 export default function Error({ error, reset }) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)